client/swagger.yaml

```diff
@@ -398,6 +398,56 @@ paths:
           description: Internal server error
```

Comment:
How about copying https://github.com/firecracker-microvm/firecracker/blob/v0.23.0/src/api_server/swagger/firecracker.yaml as is? Would it bring other unneeded changes?

Reply:
I left out the parts relevant to snapshot load and [tracking dirty pages](https://github.com/firecracker-microvm/firecracker/blob/master/src/api_server/swagger/firecracker.yaml#L869). I was not going to include snapshot load support yet as per our [discussion](https://github.com/firecracker-microvm/firecracker-containerd/pull/448) so I decided to leave the relevant API definitions altogether. That being said, copying as-is would not interfere in any way. It is perhaps best to copy as-is so that the swagger file does not need to be updated for version 0.23.0 of the SDK.

Reply:
Yeah. It is hard to track what we've copied and what we've left out. Tracking tasks can be done in GitHub issues or somewhere else, rather than this file.

Reply:
I have copied over the swagger file from firecracker v0.23.0 and updated the API.